homeassistant/components/accuweather/sensor.py

```diff
@@ -300,8 +300,10 @@ class AccuWeatherSensorDescription(
     ),
     AccuWeatherSensorDescription(
         key="PressureTendency",
+        device_class=SensorDeviceClass.ENUM,
         icon="mdi:gauge",
         name="Pressure tendency",
+        options=["falling", "rising", "steady"],
```

Comment:
Does it not need strings.json updates?

Reply:
I think everything is in place

https://github.com/home-assistant/core/blob/924cbb590f0c7d267bc321d903d4ec9b9ab38649/homeassistant/components/accuweather/strings.json#L25-L35

Reply:
Sorry - I was just surprised not to see it in the same PR!